q2_chemistree/_hierarchy.py

```diff
@@ -42,12 +46,12 @@ def make_hierarchy(collated_fingerprints: biom.Table,
     table = collated_fingerprints.to_dataframe()
     if table.shape == (0, 0):
         raise ValueError("Cannot have empty fingerprint table")
-    if not 0 <= prob_threshold <= 1:
-        raise ValueError("Probability threshold is not in [0,1]")
-    for col in table:
-        table[col] = [1 if val > prob_threshold else 0 for val in table[col]]
-    distmat = pairwise_distances(X=table, Y=None, metric='jaccard')
-    distsq = squareform(distmat)
+    if prob_threshold:
```

Comment:
This needs to be more specific about what it is testing. For example if `if prob_threshold is not None`.